clippy_lints/src/large_enum_variant.rs

```diff
@@ -0,0 +1,81 @@
+//! lint when there are large variants on an enum
+
+use rustc::lint::*;
+use rustc::hir::*;
```

Comment:
In my defense, this does not appear in either the commit info (“@foo commented with @bar”), the first commit message, the blame info (you don't seem to have used `git rebase`?), or the PR's comment (“obsoletes #xyz” implies this is more than just a rebase).